ArmPkg/ArmPkg.dsc

```diff
@@ -137,6 +137,7 @@
   ArmPkg/Drivers/CpuPei/CpuPei.inf
   ArmPkg/Drivers/ArmGic/ArmGicDxe.inf
   ArmPkg/Drivers/ArmGic/ArmGicLib.inf
+  # ArmPkg/Drivers/ArmPsciMpServicesDxe/ArmPsciMpServicesDxe.inf    # MU_CHANGE: Only build MP driver for AARCH64

```

Comment:
why the commented-out version here?

Reply:
Edk2 patch placed this entry here. But some functions this driver use are `AARCH64` only. So I commented out the original line and moved it to the AARCH64 section in the dsc file.